rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -898,6 +898,30 @@ void getActiveFederationCreationBlockHeight_afterRSKIP186AndNoValueInStorage_ret
 
     }
 
+    @Test
+    void setActiveFederationCreationBlockHeight() {
```

Comment:
No tests to remove from BridgeStorageProviderTest?

Reply:
No.